test/suite/hook_tiered.py

```diff
@@ -34,16 +34,24 @@
 #
 # Substitute tiered tables for regular (row-store) tables in Python tests.
 #
-# These hooks can be used to run the existing cursor tests on tiered tables.
-# They identify tests that create row-store tables and create tiered tables
-# instead.  The hook takes an optional argument to specify how many tiers
-# to create. The default is 2.
+# These hooks are intended for basic testing of tiered tables.  There are several
+# pieces of functionality here.
 #
-# To run with 3 tiers per table:
-#     ../test/suite/run.py --hooks tiered=3 cursor
+# 1. We add tiered storage parameters to the config when calling wiredtiger_open()
+#
+# 2. If we create an object that is *not* a table:, we add options to its config
+#    so that it will be stored local-only.  Tiered storage isn't intended (yet?) for
+#    use with LSM or colstore.
+#
+# 3. We add a call to flush_tier() after each call to checkpoint().
+#
+# 4. We stub out some functions (currently just drop) that aren't supported by tiered
```

Comment:
What about other schema operations like rename or alter? I am not sure alter works (properly) and I changed rename to return an error for tiered so that seems like it may need special processing.

Reply:
My strategy has been to stub out things as I run into tests that fail because of them. But you're right, it's probably better to just stub out the things we *know* don't/won't work.

Reply:
I've added stubs for altar, compact, rename, salvage, and verify.

Reply:
Thanks! The comment above should be updated to remove the parenthetical comment.